DuckDuckGo/it.lproj/Localizable.strings

```diff
@@ -1891,9 +1897,36 @@
 /* No comment provided by engineer. */
 "siteFeedback.urlPlaceholder" = "Quale sito web è danneggiato?";
 
+/* Message confirming that recovery code was copied to clipboard */
+"sync.code.copied" = "Codice di recupero copiato negli appunti";
+
+/* Caption for a button to delete Sync server data */
+"sync.delete.all.confirm.action" = "Eliminare i dati del server";
+
+/* Message for the dialog to confirm deleting Sync server data */
+"sync.delete.all.confirm.message" = "Tutti i dispositivi saranno disconnessi e i dati sincronizzati verranno eliminati dal server.";
+
+/* Title of the dialog to confirm deleting Sync server data */
+"sync.delete.all.confirm.title" = "Eliminare i dati del server?";
+
+/* Caption for a button to remove device from Sync */
```

Comment:
This seems to be not actually used

Reply:
Isn't this used in `UserText.syncRemoveDeviceConfirmAction` which is referenced in alert?

Reply:
yeah but the alert itself seems to not be used anywhere anymore